src/core/utils.js

```diff
@@ -626,3 +626,60 @@ export function getTextNodes(el, exclusions = []) {
   }
   return textNodes;
 }
+
+/**
+ * For any element, returns an array of title strings that applies
+ *   the algorithm used for determining the actual title of a
+ *   <dfn> element (but can apply to other as well).
+ * if args.isDefinition is true, then the element is a definition, not a
+ *   reference to a definition. Any @title or @lt will be replaced with
+ *   @data-lt to be consistent with Bikeshed / Shepherd.
+ * This method now *prefers* the data-lt attribute for the list of
+ *   titles. That attribute is added by this method to dfn elements, so
+ *   subsequent calls to this method will return the data-lt based list.
+ * @param {Element} elem
+ * @param {Object} args
+ * @returns {Array:String} array of title strings
+ */
+export function getDfnTitles(elem, args) {
+  let titleString = "";
+  let normText = "";
+  //data-lt-noDefault avoid using the text content of a definition
+  //in the definition list.
+  if (!elem.hasAttribute("data-lt-noDefault")) {
+    normText = norm(elem.textContent).toLowerCase();
+  }
+  if (elem.dataset.lt) {
+    // prefer @data-lt for the list of title aliases
+    titleString = elem.dataset.lt.toLowerCase();
+    if (normText !== "" && !titleString.startsWith(`${normText}|`)) {
+      // Use the definition itself, so to avoid having to declare the definition twice.
+      titleString = titleString + "|" + normText;
+    }
+  } else if (
+    elem.childNodes.length === 1 &&
+    [...elem.children].filter(el => el.matches("abbr, acronym")).length === 1 &&
```

Comment:
I would recommend `elem.querySelectorAll` or even `elem.getElementsByTagName`.

`acronym` is an obsolete element in HTML5, do we still need it? @marcoscaceres

Reply:
I found some instances of `acronym` in some `respecConfig`s and tests/specifcations in ReSpec. Not sure if we should remove its support. WDYT @marcoscaceres?